.changeset/ninety-hats-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-azure': minor
```

Comment:
This is not a breaking change right? Should be a `patch` in that case, since the package is on `0.x`.

Reply:
Do plugins not follow normal semantic versioning? https://backstage.io/docs/overview/versioning-policy#package-versioning-policy According to `semver` bumping the minor version is something that's backwards compatible and the major version is for breaking changes.

Reply:
Patch is for fixing "critical" bugs if we follow semver to my understanding. This is also according the docs and the release line: https://backstage.io/docs/overview/versioning-policy#main-release-line

Reply:
Since the package is at version `0.x` non-breaking changes result in a `patch` bump, since we reserve `minor` for breaking changes, which is compatible with semver.